### PR TITLE
feat: upgrade walletconnect ui to align to multiple connection design

### PR DESCRIPTION
**Pull request by woodenfurniture**
## Description

Upgrades the ui of the walletconnect dropdown to align to the multi-connection design. Does not implement multi-connections though - that will be in a follow-up.

Note that multiple address links appear as duplicates in the UI but point to different pages - this aligns to the design:
image

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

progresses #5236

## Risk

* Risk of broken walletconnect in app - connecting or disconnecting
* Risk of incorrect connection details

## Testing

* Check ability to connect+disconnect via walletconnect still works
* Check info displayed in ui is correct
* Check address links are correct

### Engineering

☝️

### Operations

☝️

## Screenshots (if applicable)

Default state (before connecting walletconnect) - unchanged from prod
image

App connected - unchanged from prod
image

Dropdown opened, default state - aligned to new design
image

Dropdown opened, collapsed state - aligned to new design. This isn't useful until multi-connection ui is implemented, but placeholder functionality is there.
image

### CODE REVIEW

**Comment by woodenfurniture** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5237** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5237" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5237?utm_source=stack-comment).

File: src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx [hunk 1]
```diff
@@ -761,27 +761,37 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
...
-        .minus(bnOrZero(state?.withdraw.estimatedGasCryptoBaseUnit))
-        .times(-1)
+      return fromBaseUnit(
+        bnOrZero(feeAssetBalanceCryptoBaseUnit)
+          .minus(bnOrZero(state?.withdraw.estimatedGasCryptoBaseUnit))
+          .times(-1),
+        feeAsset.precision,
+      )
     }
-    return bnOrZero(assetBalanceBaseUnit)
```
Comment (0xApotheosis) [inline src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx:772]: This diff looks odd?

**Review by 0xApotheosis — COMMENTED** [review]
Code looks good, except for a weird diff issue - might need to rebase.

Uppercase "address" below:

Screenshot 2023-09-11 at 9 32 28 am

File: src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx (hunk 1 above)
Comment (woodenfurniture) [reply src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx]: correct, needed a rebase ✅

**Review by 0xApotheosis — APPROVED** [review]

**Pull request merged**